CHANGELOG.md

```diff
@@ -127,7 +127,25 @@ binary | sha1 hash | md5 hash
 
 ## Known Issues and Important Steps before Upgrading
 
-* *Instructions coming soon*
+The following versions of Docker Engine are supported - *v1.10*, *v1.11*
+Although *v1.9* is still compatible, we recommend upgrading to one of the supported versions.
+
+### kubernetes Core Known Issues
```

Comment:
Yeah good idea. Except for the first known issue, I can't find a workaround for the rest though.